firefox-ios/Client/TabManagement/TabWebViewPreview.swift

```diff
@@ -99,14 +114,18 @@ final class TabWebViewPreview: UIView, ThemeApplicable {
         }
     }
 
-    func setScreenshot(_ image: UIImage?) {
-        webPageScreenshotImageView.image = image
+    private func setStackViewsVisibility(by searchBarPosition: SearchBarPosition) {
```

Comment:
maybe add a comment here on the requirements? what qualifies in being the bottom stack view or top stack view?